docs/getting-started.md

```diff
@@ -4,8 +4,8 @@
 
 To contribute to this plugin, you need the following tools installed on your computer:
 
-- [PHP](https://www.php.net/) - version 7.2 or higher, preferably installed via [Homebrew](https://brew.sh/)
-- [Composer](https://getcomposer.org/) (PHP package manager) - to install PHP dependencies. Preferably version 1.x (see #5083 for Composer v2 support).
+- [PHP](https://www.php.net/) - version 7.2 or higher, preferably installed via [Homebrew](https://brew.sh/).
```

Comment:
In my testing, it didn't work in PHP 8.0. We should document this here. 

<img width="988" alt="Screenshot 2021-02-18 at 10 52 08" src="https://user-images.githubusercontent.com/237508/108346527-62893a80-71d7-11eb-9eaf-401d760b6869.png">

Reply:
That's unrelated to this PR?

PHP-Scoper will fail on PHP 8 also on `main`

Reply:
We should just update PHP-Scoper to 0.14.0, which has PHP support.